htwoo-core/src/_patterns/atoms/loading/spinner.hbs

```diff
@@ -0,0 +1,4 @@
+<div class="hoo-spinner-xsmall"></div>
+<div class="hoo-spinner-small"></div>
+<div class="hoo-spinner-medium"></div>
+<div class="hoo-spinner-large"></div>
```

Comment:
Just show the default one. Add a markdown that it equals the medium spinner.

One thing we could do in case of the spinner have something like this:

```css
.hoo-spinner{
    
    &.xsmall{
    }

    &.small{
    }
    
     &.large{
    }
    
}
```

So default spinner with modifier classes.

Reply:
Actually, I think I tried modifier classes at first but found that `.medium` collided with some other already existing style. Because of that I went with the appoarch in the PR. It also seemed to be more in line with other styles such as `.hoo-button, .hoo-button-primary`, rather than `.hoo-button.primary`.

But if you prefer the style in your example we can go with that of course.

Reply:
There as non scoped `.medium` selector in pattern-scaffolding.scss.

Reply:
Not anymore and it was not part of htwoo anyways. scoped it now to body.

Reply:
Thanks. I pushed changes that use modifier classes instead of one class per size.

Reply:
Thank you will check it out and merge it for the version.